extensions/adhan-time/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Adhan time Changelog
 
+## [Update] - {PR_MERGE_DATE}
```

Comment:
please do not forget this @menisy

Reply:
The date gets filled in automatically based on this placeholder.

Reply:
great my bad 🙏